docs-ref-conceptual/authenticate-azure-cli-interactively.md

```diff
@@ -13,11 +13,79 @@ ms.custom: devx-track-azurecli
 
 # Sign in interactively with Azure CLI 
 
-Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login) command, which is useful for ad-hoc management tasks and for environments that require manual sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies access for script testing, learning, and on-the-fly management without needing to preconfigure service principals or other noninteractive authentication methods.
+Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login
+with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login)
+command, which is useful for ad-hoc management tasks and for environments that require manual
+sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies
+access for script testing, learning, and on-the-fly management without needing to preconfigure
+service principals or other noninteractive authentication methods.
+
+## Prerequisites
+
+* [Install the Azure CLI](./install-azure-cli.md)
+
+## Interactive login
+
+To sign in interactively, use the [az login](/cli/azure/reference-index#az-login) command. Beginning
+with Azure CLI version [2.61.0](./release-notes-azure-cli.md#may-21-2024), Windows systems use Web
+Account Manager (WAM), and Linux and macOS systems use browser-based login by default.
+
+```azurecli-interactive
+az login
+```
+
+## Subscription selector
```

Comment:
This section for **subscription selector** lacks the information on how to disable it.

https://github.com/Azure/azure-cli/pull/28910#issue-2283053031 shows how to disable subscription selector:

```
az config set core.login_experience_v2=off
```

Reply:
fixed in PR https://github.com/MicrosoftDocs/azure-docs-cli/pull/4326/files